libsquoosh/src/codecs.ts

```diff
@@ -9,6 +9,23 @@ import { cpus } from 'os';
   hardwareConcurrency: cpus().length,
 };
 
+interface DecodeModule extends EmscriptenWasm.Module {
+  decode: (data: Uint8Array) => any;
```

Comment:
I think we know the return value of the `decode` which is ImageData. So, we can use ImageData here as the return type, wdyt?

Reply:
+1, decoders will always return `ImageData`
```suggestion
  decode: (data: Uint8Array) => ImageData;
```

Reply:
I tried that but there seems to be a mismatch with the ImageData in this file and the one in `index.ts`

```ts
interface DecodeModule extends EmscriptenWasm.Module {
  decode: (data: Uint8Array) => ImageData;
}
```

![image](https://user-images.githubusercontent.com/229881/127577266-41b0a229-b1bb-4bca-b1e3-3a76067a688a.png)

Reply:
Can you try channg the
'''
ImageData: import(…)
'''

To 'ImageData: import(…).ImageData' since and then using the ImageData in decoder?

Reply:
I think I have made a mistake there while defining the global `ImageData`. We need to use this:
```
  type ImageData = import('./image_data.js').default;
```

If we do this, it works with ImageData return type above.